config/remotePlugin.js

```diff
@@ -435,6 +435,21 @@ const routeExtensions = [
     },
   },
 
+  // snapshot route
+  {
+    type: 'core.page/route',
+    properties: {
+      path: '/application-pipeline/workspaces/:workspaceName/applications/:appName/snapshots/:snapshotName',
+      exact: true,
+      component: {
+        $codeRef: 'SnapshotDetailsPage',
+      },
+    },
+    flags: {
+      required: ['HACBS', 'SIGNUP'],
+    },
+  },
+
```

Comment:
I think we need another section for activeTab, clicking on pipelineruns tab gives 404.

```js
{
    type: 'core.page/route',
    properties: {
      path: '/application-pipeline/workspaces/:workspaceName/applications/:appName/snapshots/:snapshotName/:activeTab',
      exact: true,
      component: {
        $codeRef: 'SnapshotDetailsPage',
      },
    },
    flags: {
      required: ['HACBS', 'SIGNUP'],
    },
  },
```

Reply:
It is part of the next PR